src/core/reducers/installations.js

```diff
@@ -13,10 +14,12 @@ import {
   UNINSTALL_COMPLETE,
   acceptedInstallTypes,
 } from 'core/constants';
+import log from 'core/logger';
 
 
 export default function installations(state = {}, { type, payload }) {
   if (!acceptedInstallTypes.includes(type)) {
+    log.info(`${type} not in acceptedInstallTypes so action is being ignored.`);
```

Comment:
Yeah, it definitely doesn't fit the mold of what we normally do. I think it had more setup that I wanted to guard against executing unnecessarily before but it looks pretty safe now.

The check could be removed along with the `/* instalbul ignore else */` comment and then the else could just return state by the looks of it.